backend/api/recommender.py

```diff
@@ -542,16 +546,7 @@ def recommendations_from_ml(logs, user_id, from_dt=None, to_dt=None):
     has_recs = check_recs_now(recs_week)
     rec_list = []
     if has_recs:
-        for rec in has_recs:
```

Comment:
@bhuvy2 Kinda wanted these to return the recs from the models directly, instead of the dict so I can use the unique id, and fetch existing feedback.

Okay if I do this?

Reply:
As long as it works downstream that works for me.